storage/memory/memory.go

```diff
@@ -463,27 +515,28 @@ func (m *memory) PredicatesForObject(ctx context.Context, o *triple.Object, lo *
 	if prds == nil {
 		return fmt.Errorf("cannot provide an empty channel")
 	}
+
 	oUUID := UUIDToByteString(o.UUID())
 	m.rwmu.RLock()
 	defer m.rwmu.RUnlock()
 	defer close(prds)
 
 	if lo.LatestAnchor {
-		lastTA := make(map[string]*time.Time)
-		trps := make(map[string]*triple.Triple)
-		for _, t := range m.idxO[oUUID] {
-			p := t.Predicate()
-			ppUUID := p.PartialUUID().String()
-			if p.Type() == predicate.Temporal {
-				ta, err := p.TimeAnchor()
-				if err != nil {
-					return err
-				}
-				if lta := lastTA[ppUUID]; lta == nil || ta.Sub(*lta) > 0 {
-					trps[ppUUID] = t
-					lastTA[ppUUID] = ta
-				}
-			}
+		if lo.FilterOptions != nil {
+			return fmt.Errorf("cannot have LatestAnchor and FilterOptions used at the same time inside lookup options")
```

Comment:
fmt.Error since there is no formatting

Reply:
I thought about that when writing this line too, but as far as I looked here there is no `fmt.Error` function with this behavior in the `fmt` package.